cmd/http/server.go

```diff
@@ -137,6 +139,8 @@ func (srv *Server) Shutdown() error {
 	for {
 		select {
 		case <-shutdownTimer.C:
+			// Print all running goroutines.
+			_ = pprof.Lookup("goroutine").WriteTo(os.Stderr, 1)
```

Comment:
Can't we print in a file iunder /tmp/ instead ? This looks like an output that can worry our users.

Reply:
Actually this will always happen when there are some ongoing connections from s3 clients.

Reply:
@vadmeste agree we can push it some file as necessary

Reply:
This is on shutdown only. We already print an error, so we are not worrying the user much more than we already do.

But if you'd like I can write to the temp dir.

Reply:
@klauspost the error that is printed is informative, but I think goroutines stack traces will seem more like a crash for our users. Most likely we will get a lot of inquiries in our Slack channel about that.

Reply:
@klauspost @vadmeste is right since it will print a backtrace in normal uses it will unnecessarily alarm the user.

I guess writing to temp dir is better in such situations and avoid clobbering the screen.

Reply:
No problem I'll modify it for that 👍